Sources/Paywalls/PaywallData.swift

```diff
@@ -15,6 +15,25 @@
 
 import Foundation
 
+/// Represents countries where currencies typically have zero decimal places
+public struct ZeroDecimalPlaceCountries: Codable, Sendable, Hashable, Equatable {
+
+    /// Storefront country codes that should typically display zero decimal places
+    public var apple: Set<String>
+
+    /// Storefront country codes that should typically display zero decimal places.
+    public init(apple: Set<String>) {
+        self.apple = apple
+    }
+
+    // swiftlint:disable:next missing_docs
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.apple = try container.decodeIfPresent(Set<String>.self, forKey: .apple) ?? []
+    }
+
+}
+
```

Comment:
Can this struct live inside of `PaywallData` like other structs do? We don't necessarily want `ZeroDecimalPlaceCountries` to be a thing easily available at the top of our `RevenueCat` module namespace 🤷‍♂️ 

```swift
extension PaywallData {
    public struct ZeroDecimalPlaceCountries {
        // All the good stuff here
    }
}

Reply:
Done!